File: backend/BudgetToWealthApi/Models/Holdings/Holding.cs
public class Holding : BaseEntity
{
    public required string Name { get; set; }
    public required HoldingType Type { get; set; }
    public required Guid HoldingCategoryId { get; set; }
    public HoldingCategory? HoldingCategory { get; set; }
    public string? Institution { get; set; }
    public string? UserId { get; set; }
}

File: frontend/src/app/lib/models/net-worth/Holding.ts
import { ListTableItem } from "@/app/ui/components/table/ListTable";
import { HoldingType } from "./HoldingType";
import { Category } from "../Category";

export type Holding = ListTableItem & {
  id?: number;
  name: string;
  type: HoldingType;
  holdingCategoryId: string;
  holdingCategory?: Category;
  institution?: string;
}
